src/tdi_table_data.cpp

```diff
@@ -134,11 +134,6 @@ tdi_status_t TableData::getValue(const tdi_id_t & /*field_id*/,
   return TDI_NOT_SUPPORTED;
 }
 
-tdi_status_t TableData::actionIdGet(tdi_id_t *act_id) const {
```

Comment:
Why we delete actionIdGet from the TableData?

Reply:
Made it simpler. Earlier we were returning TDI_SUCCESS every time. Now we just return the action ID directly

Reply:
ok. Then I need to change my calling function ...